packages/jest-snapshot/src/types.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
```

Comment:
Can you add the full thing? This file will be included in the distribution. The short one is great for tests etc that are not distributed 🙂